docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Gatsby For E-commerce
+---
+
+Businesses selling products online typically need a variety of software to support their experience. At a minimum, their website needs product pages, product catalog navigation, a shopping cart, and checkout. 
+
+Most have additional functionality like customer account creation, discount codes, customer reviews, tax calculation, user tracking via analytics, and content personalization.
+
+The website functions because these front-end capabilities integrate with a wide swath of software on the backend, such as inventory management, order fulfillment, business analytics, and customer engagement via email. 
+
+### Choosing a main e-commerce platform
+
+Most businesses choose a central e-commerce platform as their source of truth for these functionality. Some businesses choose to run entirely on these platforms, which can be quicker to get started with but lock you in to their choices for website creation (eg Liquid Templates for Shopify). 
+
+If you're using or considering Gatsby, your organization is likely optimizing for specific properties of your website (such as performance, development environment, design, user experience) and are therefore considering a "JAMstack", "decoupled" or "content mesh" architecture. 
+
+In this case, your options are likely between a technologically forward e-commerce vendor with headless capabilities, such as Shopify or BigCommerce, or a new, specialized JAMStack e-commerce vendor, like Snipcart, Nacelle, or CommerceLayer. All of these have out-of-the box Gatsby integrations.
+
+### Choosing additional content systems
+
+Once an organization selects a main e-commerce platform, it may also want or need other content stores that will get pulled into the website. They might choose a CMS like Contentful for complex content modelling, or Wordpress for blog content authoring. They might choose Yotpo for customer reviews or Salsify to store product SKUs. 
+
+### Why organizations build e-commerce sites with Gatsby
+
+Organizations looking for a JAMStack e-commerce site that go with Gatsby typically do so for a combination of three reasons: they want to embrace a modern development framework (React), they want to optimize performance, and they want a web framework that’s already integrated with the systems they’re using (via Gatsby’s plugin system and pre-built integrations). 
+
+From a conversion perspective, Gatsby’s lightning-fast performance is a huge win: Gatsby automates code splitting, image optimization, inlining critical styles, lazy-loading, prefetching resources, and more to ensure your site is fully optimized. 
+
+And with Gatsby’s pre-built integrations, it can pull data in from all of these sources (Shopify, plus Wordpress, Contentful, Yotpo, etc) and make it available for the React components. 
+
+### Specific e-commerce development considerations
```

Comment:
I think other important considerations are:
1. ADA compliance - Huge for every ecom brand today - it's the new ambulance chasing
2. An inventory of all commerce platform apps/plugins that impact the experience layer, and what the strategy is for making them work with Gatsby. Not a major problem area, but one that needs careful planning.

Reply:
Good point. #2 is something we're working on internally. I remember you mentioned #1 to me before, thinking about how to work this in. Probably on an accessibility page and then link it in. Do you have any stats on increased litigation around accessibility issues on websites?